docs/servable-types.rst

```diff
@@ -112,7 +112,30 @@ can be created from this model file and names for the output classes:
 
 .. code-block:: python
 
-    model info = KerasModel.create_model('model.h5', ["y"])
+    model_info = KerasModel.create_model('model.h5', ["y"])
+
+Models with weights and architecture as separate files can be described using:
+
+.. code-block:: python
+
+	model_info = KerasModel.create_model('model.h5', ["y"], arch_path='arch.json')
+
+Keras also allows users to add their own custom layers to there models for any custom operation
+that has trainable weights. Use this when the Keras Lambda layer does not apply. In Keras, 
+these layers can be added when loading the model:
+
+.. code-block:: python
+
+	model = load_model('model.h5', custom_objects={'CustomLayer': CustomLayer})
+
+Adding custom layers to a DLHub description can be achived with the add_custom_object method, which takes the name
```

Comment:
Put "add_custom_data" inside of two accent marks (e.g., \`\`add_custom_object\`\`) to make it display as code.